renderer/chrome.go

```diff
@@ -63,18 +63,16 @@ func listenForResponse(c chan *network.ResponseReceivedReply, responseReceivedCl
 
 func CreatePdf(ctx context.Context, params ChromeParameters) ([]byte, []byte, error) {
 	// Use the DevTools API to manage targets
-	devt := devtool.New("http://127.0.0.1:9222")
-	pt, err := devt.Get(ctx, devtool.Page)
+	devt, err := devtool.New("http://127.0.0.1:9222").Version(ctx)
 	if err != nil {
-		pt, err = devt.Create(ctx)
-		if err != nil {
-			return nil, nil, err
-		}
+		return nil, nil, err
 	}
-	defer devt.Close(ctx, pt)
+
+
+
```

Comment:
Clean up

Reply:
Just confirming by "Clean up" you're referring to white space / formatting?

Reply:
yes